tests/storage/exception_test.py

```diff
@@ -77,6 +84,25 @@ def test_info():
         }
 
 
+def test_VolumeGroupDoesNotExist():
+    # Require a VG name or UUID at initialization.
+    # Empty constructor shall raise.
+    with pytest.raises(ValueError):
+        e = storage_exception.VolumeGroupDoesNotExist()
+
+    # Expected error type is LVMCommandError.
+    with pytest.raises(TypeError):
+        e = storage_exception.VolumeGroupDoesNotExist("vg", error="error")
+
+    # Correct initialization, test info.
+    lvm_error = storage_exception.LVMCommandError(0, 1, 2, 3)
```

Comment:
I would make this more real:

    fake_error = storage_exception.LVMCommandError(
        rc=5, command=["fake"], out=["fake output"], err=["fake error"])